mchf-eclipse/drivers/audio/cw/cw_gen.h

```diff
@@ -16,15 +16,14 @@
 
 #include "arm_math.h"
 
-// Exports
 void    CwGen_Init();
 
 void    CwGen_PrepareTx();
 void    CwGen_SetSpeed();
 
-bool    CwGen_TimersActive();
 bool    CwGen_Process( float32_t *i_buffer, float32_t *q_buffer, uint32_t size );
 
+// There is no real needs to call these functions. Going remove after adding more tests.
 void    CwGen_DahIRQ();
```

Comment:
What do you mean with this statement? These functions are called from the interrupt handler for the PTT/paddle GPIOs, see https://github.com/df8oe/UHSDR/blob/def4f73f1af6d081b2fb43cffb458f312f4fa6ed/mchf-eclipse/src/uhsdr_main.c#L62.

Reply:
> What do you mean with this statement? These functions are called from the interrupt handler for the PTT/paddle GPIOs, see
> 
> https://github.com/df8oe/UHSDR/blob/def4f73f1af6d081b2fb43cffb458f312f4fa6ed/mchf-eclipse/src/uhsdr_main.c#L62
> 
> .

I know, but answer why and for what exactly?